recipe_scrapers/tasty.py

```diff
@@ -0,0 +1,28 @@
+from ._abstract import AbstractScraper
+
+
+class Tasty(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "tasty.co"
+
+    def title(self):
+        return self.schema.title().replace(" Recipe by tasty", "")
```

Comment:
I'm conflicted by this replacement. On one hand the suffix should be left alone to be accurate to the source, but on the other it's just clutter to the recipe.

In such cases, the only source of truth that we have is the source data and thus I think it it should be left in. Users can always remove it later.

Reply:
Was being a bit selfish and coding it how i would want to see the data.  Totally agree though the source of truth should be the data source.